docs: update Flex manual to remove quick transfer from ODD (#21017)
OpenTrons/opentrons-api: docs/flex/docs/touchscreen/protocol-details.md

- **Author:** Who created the protocol.

> - **Description:** For protocols with long descriptions, scroll to read the full text. The protocol description also tells you if your protocol is a quick transfer.

Reviewer: Repetition: "protocol"

```suggestion
- **Description:** Scroll to read the full text. Also identifies quick transfer protocols.
```

Author: thanks!
